lib/src/sdk_api/view/video_view.dart

```diff
@@ -161,6 +159,7 @@ class _VideoViewState extends State<VideoView> {
   @override
   Widget build(BuildContext context) {
     final Map<String, String> creationParams = {};
+    final Map<String, String> videoViewParams = {};
```

Comment:
`creationParams` was supposed to contain all the parameters required for the creation of the view. Perhaps if we call only one method to initialise the native view we can achieve this. Please see the comments stating "Calling 2 methods here is not ideal."